docs/manage-tags-in-gravitino.md

```diff
@@ -0,0 +1,328 @@
+  "updates": [
+    {
+      "@type": "rename",
+      "newName": "tag2"
+    },
+    {
+      "@type": "updateComment",
+      "newComment": "This is an updated tag"
+    },
+    {
+      "@type": "setProperty",
+      "property": "key3",
+      "value": "value3"
+    },
+    {
+      "@type": "removeProperty",
+      "property": "key1"
+    }
+  ]
+}' http://localhost:8090/api/metalakes/test/tags/tag1
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+GravitinoClient client = ...
+Tag tag = client.alterTag(
+    "tag1",
+    TagChange.rename("tag2"),
+    TagChange.updateComment("This is an updated tag"),
+    TagChange.setProperty("key3", "value3"),
+    TagChange.removeProperty("key1"));
+```
+
+</TabItem>
+</Tabs>
+
+Currently, Gravitino support the following tag changes:
+
+| Supported modification    | JSON                                                         | Java                                      |
+|---------------------------|--------------------------------------------------------------|-------------------------------------------|
+| Rename a tag              | `{"@type":"rename","newName":"tag_renamed"}`                 | `TagChange.rename("tag_renamed")`         |
+| Update a comment          | `{"@type":"updateComment","newComment":"new_comment"}`       | `TagChange.updateComment("new_comment")`  |
+| Set a fileset property    | `{"@type":"setProperty","property":"key1","value":"value1"}` | `TagChange.setProperty("key1", "value1")` |
+| Remove a fileset property | `{"@type":"removeProperty","property":"key1"}`               | `TagChange.removeProperty("key1")`        |
+
+### Delete a tag
+
+You can delete a tag by its name.
+
+<Tabs groupId='language' queryString>
+<TabItem value="shell" label="Shell">
+
+```shell
+curl -X DELETE -H "Accept: application/vnd.gravitino.v1+json" \
+http://localhost:8090/api/metalakes/test/tags/tag2
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+GravitinoClient client = ...
+client.deleteTag("tag2");
+```
+
+</TabItem>
+</Tabs>
+
+## Tag associations
+
+Gravitino allows you to associate and disassociate tags with metadata objects. Currently, only
+`CATALOG`, `SCHEMA`, `TABLE`, `FILESET`, `TOPIC` objects can be tagged.
+
+### Associate and disassociate tags with a metadata object
+
+You can associate and disassociate tags with a metadata object by providing the object type, object
+name and tag names.
+
+<Tabs groupId='language' queryString>
+<TabItem value="shell" label="Shell">
+
+```shell
+curl -X POST -H "Accept: application/vnd.gravitino.v1+json" \
+-H "Content-Type: application/json" -d '{
+  "tagsToAdd": ["tag1", "tag2"],
+  "tagsToRemove": ["tag3"]
+}' http://localhost:8090/api/metalakes/test/tags/catalog/catalog1
+
+curl -X POST -H "Accept: application/vnd.gravitino.v1+json" \
+-H "Content-Type: application/json" -d '{
+  "tagsToAdd": ["tag1"]
+}' http://localhost:8090/api/metalakes/test/tags/schema/catalog1.schema1
```

Comment:
We'd better add some description about how to construct the URL, as the value `catalog1.schema1` varies according to different objects.